teleport/dev/roles/developer.yaml

```diff
@@ -0,0 +1,74 @@
+kind: role
+metadata:
+  name: developer
+spec:
+  allow:
+    app_labels:
+      role: developer
```

Comment:
did you try appliyng only this:
![grafik](https://github.com/hawk-ai-aml/github-actions/assets/104775085/3f8b883b-7882-49ef-8d8e-583fc2cda67b)
could be enough, as we are not using any other functionality right now

Reply:
yes i tried on dev